### DISCUSSION TITLE
Perhaps not supporting Exceptions is one of your biggest mistake

### DISCUSSION THREAD

**Discussion by ghost** [objeck/objeck-lang#386] [General]
Look at how ugly this piece of code is:

https://github.com/objeck/objeck-lang/blob/master/programs/examples/json_file.obs

You have to check if it's not `Nil` all the time. There are too many `if` statements that make it look ridiculous. If there is no need to check if it's `Nil` or not, a simple `select..label` is much better.

**Comment by objeck** [comment]
Thanks, I’m shutting down the project. The nested statements can be minimized by checking for inverse values. With exceptions, how do you know what failed and where? I'm tired and care little about ugly code right now. I will cancel the project. Thank you for your feedback.
